### ISSUE TITLE
Same posters for different movies

### ISSUE THREAD

**Issue by Frankdoc** [boxeehacks/boxeehack#537]
I updated my movie library using ViMediaManager. It created nfo files and added posters and fanart etc. I rescanned my folders in Boxee Box(+ hack) but I notice about 15 movies that all have the same poster of 1 movie. When I re-identify using NFO the poster remains wrong. When I identify the movie using manual search, it does get the correct poster. When I check the movie folder the file poster.jpg is correct.

What's wrong? Fix? Deleting all the nfo files and let boxee box identify it like it wold without nfo?

**Comment by michske86** [comment]
I've had the same problem and haven't had any luck with re-adding, re-scanning,... the problem consists. 
I've had the issue in the beginning of boxee+ as well, and it got automatically resolved with re-adding en scanning then, but this doesn't seem to help anymore since i had it a second time...

**Comment by Frankdoc** [comment]
Hmm damn.. I cleared the cache as well multiple times, but no luck either. When I manually identified the movie it did work. So maybe a solution would be is to delete the source, then delete the nfo's created by vimediamnager, and then re add the source. Then boxee box will manually identify the movies. But such a hassle..

**Comment by michske86** [comment]
Letting the boxee box identify would be hell, since I'm not impressed at all with the automatic identification... hence the nfo's :)
You can delete the nfo's and then remake them from inside vimediamanager though... so deleting the sources, deleting the nfo's, then remaking the nfo's and then readding the sources might be helpful... maybe :)

**Comment by Frankdoc** [comment]
Yeah the internal system isn't the greatest, although it has worked pretty well for me with a few exceptions. Hmm I don't know what could be wrong with the nfo's though, they were correctly identified in the program, and the folder.jpg is also the correct poster in the folder. Anyways I re added the source now without changing any files, let's see what that gives.
Btw is it normal that identifying has become painfully slow when you have all those nfo's (per episode of a show). Might delete them, it's insanely slow

**Comment by Frankdoc** [comment]
Actually made it worse by re adding the source, now most of my movies have the same poster:/ I hope this bug gets heared here. Have uninstalled boxeeplus now and will re-add the source. If it's still the same problem I'll delete those nfo's.
*edit: went into windows explorer, searched for .nfo and deleted all of them. then re added the source and now everything is okay again. Some movies had to be identified manually because of bad filenames, but at least the posters are okay.  I have the same problem for some series as well but I'll just leave that be, it's only 3 series with wrong posters..

**Comment by jonathanq** [comment]
Just wanted to add - I had the same issue (also using ViMediaManager).  I tried a bunch of different things (clearing media thumbnails etc).  However what I _think_ fixed it was this:
- Make sure ViMediaManager is creating the correct image file (folder.jpg - at some point it stopped doing that and was creating poster.jpg)
- (once folder.jpg exists) Select the movies with the wrong posters, and go to "Identify" on the Boxee and tell it to use the NFO file

If I remember correctly, that was the trick that finally fixed it for me.  Hope it helps, I spent a lot of time trying a bunch of things to solve that problem.  So thats why I am not 100% those were the steps.
